src/main/java/it/gov/pagopa/admissibility/service/onboarding/AdmissibilityEvaluatorMediatorServiceImpl.java

```diff
@@ -58,21 +68,18 @@ public void execute(Flux<Message<String>> messageFlux) {
                 .subscribe(evaluationDTO -> log.info("[[ADMISSIBILITY_ONBOARDING_REQUEST]] Processed offsets committed successfully"));
     }
 
-    private Mono<EvaluationDTO> executeAndCommit(Message<String> message) {
+    private Mono<ImmutableTriple<EvaluationDTO, InitiativeConfig,OnboardingDTO>> executeAndCommit(Message<String> message) {
```

Comment:
if we think to EvaluationDto as a base model extended by the model sent to onboardingOutcome and rankingRequest systems, we could just return it